.github/workflows/publish_sdm_connector.yml

```
@@ -0,0 +1,164 @@
+    environment:
+      name: DockerHub
+      url: https://hub.docker.com/r/airbyte/source-declarative-manifest/tags
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Detect Version (release tag)
+        if: startsWith(github.ref, 'refs/tags/v')
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:6:39: Double quote to prevent globbing and word splitting

(shellcheck)